integrations-and-supported-tools/catalyst/scripts/run_examples.sh

```diff
@@ -5,6 +5,3 @@ pip install -U -r requirements.txt
 
 echo "Running Neptune_Catalyst.py..."
 python Neptune_Catalyst.py
-
```

Comment:
Could you help me understand why the more options script was removed?

Reply:
it was a duplicate of the quickstart script